examples/react/playground/src/stories/AuthenticationSubmitted.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import { AuthenticationSubmitted } from '@w3ui/react'
```

Comment:
musing: perhaps we should make the naming consistent here so that we have `ComponentName` and `ComponentNameStateName`
```suggestion
import { AuthenticatorSubmitted } from '@w3ui/react'
```
like
- `SpaceCreator` -> `SpaceCreatorCreating`

or perhaps we should export a "view(s) only" component, that takes a state prop that we can set from a single story per Component, so our docs pages show all the things... tho that would force us to make a component that takes all the needed state and handlers for each of it's states, so perhaps the current pattern is better for now.

Reply:
yeaaaaa - I'm not totally happy with the naming/structure here either, but similarly uncertain on the best "API design" here - I'll leave it as it is, but am doing a pass getting ready for Adam to come in and do some design work and will keep this in mind as I do, thanks!